src/partials/Banner.svelte

```diff
@@ -0,0 +1,93 @@
+<script>
```

Comment:
The banner partial looks great. Feels like it should cover all our pages  and useful for any future content. Would we use this partial in individual Tools?

Reply:
Ah right, I missed the usage of it in the Tools pages because they have their own Header partial. We could use it there as well.

Reply:
@mukhtyar I spent a little bit of time fussing with swapping the Tool Header with the Banner. There are a few subtle differences that make it a bit tricky and require overrides but it could potentially work.